backend/src/main/java/de/neuefische/backend/controller/LoginController.java

```diff
@@ -37,7 +37,7 @@ public String login(@RequestBody LoginData data) {
                     new UsernamePasswordAuthenticationToken(data.getName(), data.getPassword())
             );
             LOG.info("User: " + data.getName().toUpperCase() + " ist eingeloggt");
-            return jwtService.createToken(new HashMap<>(), data.getName());
+            return jwtService.createToken(new HashMap<>(), data.getName(), data.getTimeout());
```

Comment:
Validate the experation time. A too high number might be a security vulnerability.

Reply:
good point thank you! I opened a new issue in the project plan for this